op-chain-ops/genesis/testdata/test-deploy-config-full.json

```diff
@@ -0,0 +1,47 @@
+{
+  "l1StartingBlockTag": "earliest",
+  "l1GenesisBlockTimestamp": "0x1234",
+  "l1ChainID": 901,
+  "l2ChainID": 902,
+  "l2BlockTime": 2,
+  "maxSequencerDrift": 10,
+  "sequencerWindowSize": 4,
+  "channelTimeout": 30,
+  "p2pSequencerAddress": "0x9965507d1a55bcc2695c58ba16fb37d819b0a4dc",
+  "optimismL2FeeRecipient": "0x42000000000000000000000000000000000000f0",
+  "batchInboxAddress": "0x42000000000000000000000000000000000000ff",
+  "batchSenderAddress": "0x15d34aaf54267db7d7c367839aaf71a00a2c6a65",
+  "l2OutputOracleSubmissionInterval": 6,
+  "l2OutputOracleStartingTimestamp": -1,
+  "l2OutputOracleProposer": "0x90f79bf6eb2c4f870365e785982e1f101e93b906",
+  "l2OutputOracleOwner": "0x0000000000000000000000000000000000000000",
+  "l1BlockTime": 15,
+  "l1GenesisBlockNonce": "0x0",
+  "cliqueSignerAddress": "0x3c44cdddb6a900fa2b585dd299e03d12fa4293bc",
+  "l1GenesisBlockGasLimit": "0xe4e1c0",
+  "l1GenesisBlockDifficulty": "0x1",
+  "l1GenesisBlockMixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "l1GenesisBlockCoinbase": "0x0000000000000000000000000000000000000000",
+  "l1GenesisBlockNumber": "0x0",
+  "l1GenesisBlockGasUsed": "0x0",
+  "l1GenesisBlockParentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "l1GenesisBlockBaseFeePerGas": "0x3b9aca00",
```

Comment:
I think a more simple solution would be just use numbers instead of hex strings. The go code can handle them and they will end up as the correct type by the time the `genesis.json` is serialized. If you want me to implement this in the hh tasks first to show that it can work, I am happy to do so

Reply:
Done - though some of these are bignums, and need to stay hex. The gas values are an example of this.